pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -473,7 +476,29 @@ public void run() {
           new CommandLine(this),
           "Unable to connect to multiple networks simultaneously. Specify one of --ropsten, --rinkeby or --goerli");
     }
+
     final EthNetworkConfig ethNetworkConfig = ethNetworkConfig();
+
+    // ensure all bootstrapNodes are in the whitelist
```

Comment:
I suggest removing this comment and wrapping this logic into a method with a descriptive name.